tests/device/timeout.c

```diff
@@ -32,13 +32,13 @@
 static int test_timeout(void)
 {
 	struct nvme_cqe cqe;
-	struct timespec timeout = {.tv_sec = 1};
+	uint64_t timeout_ns = 1000000000;
```

Comment:
Can we somehow keep timespec here? If the problem is in ccan/time, then we can maybe fix the include there?